lighthouse-core/report/handlebar-helpers.js

```diff
@@ -63,10 +63,9 @@ const handlebarHelpers = {
     return calculateRating(totalScore);
   },
 
-  // Converts an aggregation's score to a rating used for styling.
+  // Converts an aggregation's score to a pass/fail rating.
   getAggregationScoreRating: score => {
-    const rating = calculateRating(score * 100);
-    return rating !== 'good' ? 'poor' : rating; // avg rating maps to failure for aggregation.
+    return score === 1 ? 'good' : 'poor';
```

Comment:
What do you think all boolean audits should do?

<img width="573" alt="screen shot 2017-02-21 at 2 11 47 pm" src="https://cloud.githubusercontent.com/assets/238208/23191045/ab9be8fa-f83f-11e6-987e-55940216d202.png">

IMO, the overall icon should be orange even though the score is technically green 5/6 => 83/100.